src/domain/cartProductHandler.ts

```diff
@@ -0,0 +1,7 @@
+import { CartProduct } from '../types/product';
+
+export const findTargetProduct = (cartProducts: CartProduct[], id: number) =>
+  cartProducts.find((cartProduct) => id === cartProduct.product.id);
+
+export const deleteProduct = (cartProducts: CartProduct[], id: number) =>
+  cartProducts.filter((cartProduct) => cartProduct.product.id !== id);
```

Comment:
```suggestion
export const deleteProduct = (cartProducts: CartProduct[], id: number): CartProduct[] =>
  cartProducts.filter((cartProduct) => cartProduct.product.id !== id);
```

와 같이 해당 함수는 함수명만 보고 반환값이 무엇인지 판단하기 어려울 것 같아서, return 타입을 명시해 주어도 좋을 것 같습니다~